### PR TITLE
Updating the What is ROCm page and related content

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: docs/rocm.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+compilers, debuggers, and libraries. ROCm is fully integrated into machine learning (ML) frameworks, 
+such as PyTorch and TensorFlow. 
 
-ROCm’s goal is to allow our users to maximize their GPU hardware investment.
-ROCm is designed to help develop, test and deploy GPU accelerated HPC, AI,
-scientific computing, CAD, and other applications in a free, open-source,
-integrated, and secure software ecosystem.
+You can deploy ROCm using:
+* Bare Metal Installation (single and multi version variants)
+* Containers (such as Docker and Spack)
```
Comment (saadrahim) [inline docs/rocm.md:24]: spack is not a container, its another way to deploy. Spack is traditionally used in large clusters in the US but now has a wider audience.

Comment (saadrahim) [inline docs/rocm.md:24]: I believe we support singularity and docker for containers.

File: docs/rocm.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
-Spack, and your own build from source.
+ROCm supports programming models, such as OpenMP and OpenCL, and includes all necessary OSS 
+compilers, debuggers, and libraries. ROCm is fully integrated into machine learning (ML) frameworks, 
+such as PyTorch and TensorFlow. 
 
-ROCm’s goal is to allow our users to maximize their GPU hardware investment.
-ROCm is designed to help develop, test and deploy GPU accelerated HPC, AI,
-scientific computing, CAD, and other applications in a free, open-source,
-integrated, and secure software ecosystem.
+You can deploy ROCm using:
```
Comment (saadrahim) [inline docs/rocm.md:22]: We might want to just move this stuff to a new page. I have left a deploy page missing as I was waiting for Windows to be released before doing this.

**Review by saadrahim — DISMISSED** [review]

**Comment by saadrahim** [comment]
Can you update the README.md file as well?

**Comment by LisaDelaney** [comment]
> Can you update the README.md file as well?

Sure can.

**Review by saadrahim — APPROVED** [review]

**Pull request merged**